### PR TITLE
[READY] Desperate Assistance Battleforce

### PR DESCRIPTION
**Pull request by NopemanMcHalt**
## About The Pull Request
This silly <strike>meme-</strike>PR adds another flavor loadout that is exclusive to Assistants besides grey jumpsuit.
~~**_UPDATE_**: **Made meme poll for a different name**. Most voted will be used. Will be decided in 3 or so hours.~~ Okay poll decided it to be DAB instead of ASSU.

## Why It's Good For The Game
For sake of messing around. Suit and helmet don't provide any armor.

## Changelog
:cl: Anonymous
add: Added NEET-- I mean, DAB Suit and Helmet into loadout. Exclusive to Assistants, for obvious reasons, and don't provide any armor. Goes well with balaclava, finger-less gloves and jackboots for that true tactic~~f~~ool experience.
/:cl:

### ToDo:
- [x] Digi variant.
   - [x] Improve side sprites.
- [x] Move code and files from modular into main folders.

### Preview (NOTE: Resized improperly)
image

### CODE REVIEW

File: modular_citadel/code/modules/clothing/head/head.dm [hunk 1]
```diff
@@ -17,4 +17,13 @@
...
 
 /datum/component/storage/concrete/pockets/tiny/spacenam
-	attack_hand_interact = TRUE		//So you can actually see what you stuff in there
\ No newline at end of file
+	attack_hand_interact = TRUE		//So you can actually see what you stuff in there
+
+/obj/item/clothing/head/assu_helmet
+	name = "ASSU helmet"
+	icon = 'modular_citadel/icons/obj/clothing/assu.dmi'
+	alternate_worn_icon = 'modular_citadel/icons/mob/clothing/assu.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/clothing/head/head.dm]: Just move this item to `code/modules/clothing/head/misc.dm`, remove these two lines and add the sprites to the standard `clothing/hats.dmi` and `mob/head.dmi` instead.
It's annoying to keep track of the dozens already existing snowflake .dm and .dmi.

File: modular_citadel/code/modules/clothing/suits/suits.dm [hunk 2]
```diff
@@ -40,4 +40,17 @@
...
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
-	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = -5, "acid" = -15) //nylon sucks against acid
\ No newline at end of file
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = -5, "acid" = -15) //nylon sucks against acid
+
+/obj/item/clothing/suit/assu_suit
+	name = "ASSU suit"
+	desc = "A cheap replica of old SWAT armor. On its back, it is written: \"<i>Assistance Skipping Special Unit</i>\"."
+	icon = 'modular_citadel/icons/obj/clothing/assu.dmi'
+	alternate_worn_icon = 'modular_citadel/icons/mob/clothing/assu.dmi'
```
Comment (Ghommie) [inline modular_citadel/code/modules/clothing/suits/suits.dm]: Same, move this item to `code/modules/clothing/suits/miscellaneous.dm`, remove these two lines and add the sprites to `clothing/suits.dmi` and `mob/suit.dmi` (no need to move the digi ones) instead.

**Comment by Ghommie** [comment]
Also the left/right digi versions sprites could use a few more pixels on the legs.

**Comment by NopemanMcHalt** [comment]
Damn it, somehow digitigrade sprites keeps getting misaligned. <strike>Disregard last commit.</strike>

**UPDATE:** Oh fuck, of course it doesn't work because I forgot to remove mutantrace limitation. 🤦‍♂

**Comment by NopemanMcHalt** [comment]
Oh fuck, of course it doesn't work because I forgot to remove mutantrace limitation. 🤦‍♂

**Comment by NopemanMcHalt** [comment]
All done, I hope.

**Review by silicons — APPROVED** [review]

**Pull request merged**